### PR TITLE
Update convert-source-map typings to match it's API at 1.3.0

### PR DESCRIPTION
**Pull request by mgroenhoff**
https://github.com/thlorenz/convert-source-map/blob/v1.3.0/README.md

### CODE REVIEW

**Comment by mgroenhoff** [comment]
@zhengbli, what if @AndrewGaspar doesn't 👍 or 👎 ?

**Comment by zhengbli** [comment]
@mgroenhoff  We will go ahead merge the pr if there are no response from the reviewer for long enough.

**Comment by zhengbli** [comment]
👍

File: convert-source-map/index.d.ts [hunk 1]
```diff
@@ -1,26 +1,27 @@
-// Type definitions for convert-source-map
+// Type definitions for convert-source-map v1.3.0
 // Project: https://github.com/thlorenz/convert-source-map
 // Definitions by: Andrew Gaspar <https://github.com/AndrewGaspar/>
```
Comment (zhengbli) [inline convert-source-map/index.d.ts]: @mgroenhoff can you add your name to the author line?

**Review by zhengbli — CHANGES_REQUESTED** [review]

**Review by zhengbli — APPROVED** [review]

**Comment by mgroenhoff** [comment]
Thanks!

**Pull request merged**